dCommon/NiPoint3.inl

```diff
@@ -0,0 +1,37 @@
+#pragma once
```

Comment:
pragma once not needed for inl files, if anything the inverse is necessary
```
#ifndef NiPoint3HeaderGuard
#error "This is only to be included inline in NiPoint3.h.  Do not include directly"
#endif
```

Reply:
Would probably want a combo of the two if you wanted to make sure the inl is also only included once

Reply:
Good idea. I'll implement that in a bit.